test/misc/update-merkle-distributor-rewards-parameters.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
+import { BigNumber } from 'ethers';
+
+import { describeContract, TestContext } from '../helpers/describe-contract';
+
+// LP rewards should stay the same (1150685000000000000000000)
+// Trader rewards should be reduced by 25% (3835616000000000000000000 => 2876712000000000000000000)
+// Alpha parameter should be set to 0
+
+function init() { }
+
+describeContract('MerkleDistributor', init, (ctx: TestContext) => {
+
+  it('Merkle distributor reward parameters are updated', async () => {
```

Comment:
Possible to also verify the IPFS hash of the proposal is correct in this test? 🙏